.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,15 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
```

Comment:
what is the difference between the `about` session and the description on line 10?

Reply:
Related to previous comment